govcd/sample_govcd_test_config.json

```diff
@@ -7,7 +7,8 @@
 		"sysOrg": "System",
 		"//": "(Optional) In some cases the vCloud Director SDK must wait. It can be",
 		"//": "customized using the below var. If it is not set SDK assumes a default value.",
-		"maxRetryTimeout": 60
+		"maxRetryTimeout": 60,
+    "httpTimeout" : 600
```

Comment:
Looks to be misaligned.

Reply:
in UI shows normally

Reply:
This line uses spaces, while the others use tabs

Reply:
![image](https://user-images.githubusercontent.com/41415459/63348137-58c41400-c361-11e9-8691-924f2e87b0d3.png)
now ok